forge-gui/res/cardsfolder/upcoming/animal_friend.txt

```diff
@@ -0,0 +1,13 @@
+Name:Animal Friend
+ManaCost:1 G
+Types:Enchantment Aura
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Pump
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddTrigger$ AttackTrig | Description$ Enchanted creature has "Whenever this creature attacks, create a 1/1 green Squirrel creature token. Put a +1/+1 counter on that token for each Aura and Equipment attached to this creature other than CARDNAME."
+SVar:AttackTrig:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Whenever this creature attacks, create a 1/1 green Squirrel creature token. Put a +1/+1 counter on that token for each Aura and Equipment attached to this creature other than Animal Friend.
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_squirrel | RememberTokens$ True | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Remembered | CounterType$ P1P1 | CounterNum$ X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Count$Valid Equipment.Attached,Aura.Attached+!EffectSource
```

Comment:
!EffectSource isn't the correct param to implement "attached to this creature other than CARDNAME"

Thoughts ?

Reply:
Try `NotDefinedOriginalHost`

Reply:
Thanks working 
![Uploading image.png…]()